frontend/src/scenes/notebooks/Nodes/NotebookNodeBacklink.tsx

```diff
@@ -22,29 +20,28 @@ const ICON_MAP = {
 }
 
 const Component = (props: NodeViewProps): JSX.Element => {
-    const { shortId } = useValues(notebookLogic)
-    const { notebookLinkClicked } = useActions(notebookSidebarLogic)
-
     const type: TaxonomicFilterGroupType = props.node.attrs.type
-    const href: string | undefined = backlinkHref(props.node.attrs.id, type)
+    const href: string = backlinkHref(props.node.attrs.id, type)
```

Comment:
We don't need to specify the type here right? No big deal but no sense in over-typing if we don't need it.